### PR TITLE
Check IDR_OMERO_CONFIGURATION_URL env var for auto-configuration (IDR-0.4.4)

### PR DESCRIPTION
**Pull request by manics**
Updates the environment passed to notebooks:
- Added `IDR_OMERO_CONFIGURATION_URL`
- Added `IDR_PORT`
- Renamed `IDR_HOSTNAME` to `IDR_HOST`

See
- https://github.com/IDR/idr-notebooks/pull/39
- https://github.com/bramalingam/idr-py/pull/1

### CODE REVIEW

**Comment by joshmoore** [comment]
:+1: for merging along with https://github.com/IDR/idr-notebooks/pull/39#issuecomment-333521796 -- propose `0.5.0`?

**Comment by manics** [comment]
You can merge but hold off tagging for another couple of days

File: jupyterhub_config.py [hunk 1]
```diff
@@ -117,11 +117,10 @@
...
 
-c.Spawner.environment = {
-    'IDR_HOSTNAME': os.getenv('IDR_HOSTNAME'),
-    'IDR_USER': os.getenv('IDR_USER'),
-    'IDR_PASSWORD': os.getenv('IDR_PASSWORD'),
-}
+c.Spawner.environment = {}
+for name in ('OMERO_CONFIGURATION_URL', 'HOST', 'PORT', 'USER', 'PASSWORD'):
+    if os.getenv('IDR_' + name):
+        c.Spawner.environment['IDR_' + name] = os.getenv('IDR_' + name)
```
Comment (sbesson) [inline jupyterhub_config.py]: Assuming the `IDR_HOSTNAME` -> `IDR_HOST` renaming is done on purpose, this looks fine to me.

Comment (manics) [reply jupyterhub_config.py]: It's intended to match the variable names in https://github.com/IDR/idr-py/blob/d0d7399e3b44aa91d6f6d5d95188845e97f0f418/idr/connections.py#L9

**Pull request merged**